.travis.yml

```diff
@@ -10,11 +10,18 @@ dist: xenial
 sudo: required
 go: 1.12.x
 
+before_install:
+  - make install-mock
+  - make mocks
+
 jobs:
   include:
-    - stage: validate
+    - stage: Lint
       script: make lint
-    - script: make vet
-    - script: make test
-    - script: make coverage
+    - stage: Vet
```

Comment:
Use the previous to put it all into a single named stage, so they execute in parallel